docker/docker_tools_misc.py

```diff
@@ -23,88 +23,72 @@
 
 # Local application
 # -----------------
-from ci_utils import chdir, env, xqt
+from ci_utils import env, xqt
 
 
 # Globals
 # =======
-# The name of the container running the Runestone servers.
-res = subprocess.run(
-    'docker ps --filter "ancestor=runestone/server"  --format "{{.Names}}"',
-    shell=True,
-    capture_output=True,
-    text=True,
-)
-RUNESTONE_CONTAINER_NAME = res.stdout.strip()
```

Comment:
I left this as a global thinking it would be useful for several commands that we would like to migrate to the tools, such as accessing the database etc.

Reply:
Since this means we execute `docker` for EVERY command, I tried to avoid this where possible.

The "run `psql`" command should, IMHO, be in `rsmanage` since it applies to BookServer lite (no Docker) or the Docker setup. This is somewhat a design decision, so I'll post in the newsgroup -- it seems like a better place than here, where it's somewhat incidental to this PR.